locales/nl/nl.json

```
@@ -610,7 +610,7 @@
     "These people have been invited to your team and have been sent an invitation email. They may join the team by accepting the email invitation.": "Deze personen hebben een uitnodiging ontvangen om lid te worden van je team. Ze kunnen deelnemen door de uitnodiging te accepteren.",
     "This account does not have an active subscription.": "Dit account heeft geen actief abonnement.",
     "This action is unauthorized.": "Deze actie is niet toegestaan.",
-    "This coupon code can only be used by new customers.": "This coupon code can only be used by new customers.",
+    "This coupon code can only be used by new customers.": "Deze coupon code kan alleen gebruikt worden door nieuwe klanten.",
```

```suggestion
    "This coupon code can only be used by new customers.": "Deze couponcode kan alleen gebruikt worden door nieuwe klanten.",
```
https://onzetaal.nl/taaladvies/samenstelling